pkg/wallet/wallet.go

```diff
@@ -150,8 +150,8 @@ func (w *Wallet) Save() error {
 	return w.writeRaw(data)
 }
 
-// savePretty saves wallet in a beautiful JSON.
-func (w *Wallet) savePretty() error {
+// SavePretty saves wallet in a beautiful JSON.
+func (w *Wallet) SavePretty() error {
```

Comment:
If we're doing this, we don't really need both `Save` and `SavePretty` then.

Reply:
Fixed.